src/main/java/com/example/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,10 +1,24 @@
 package com.example.bookshop.dto;
 
+import com.example.bookshop.validation.Isbn;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import org.hibernate.validator.constraints.Length;
+
 import java.math.BigDecimal;
 
-public record CreateBookRequestDto(String title,
+public record CreateBookRequestDto(@NotNull
+                                   @Length(max = 250)
+                                   String title,
+                                   @NotNull

```

Comment:
it is better to use `@NotEmpty` with strings


it will fail validation if string is empty or blank